js/routes/MeteorRoute.js

```diff
@@ -1,10 +1,28 @@
 const express = require('express')
 const router = express.Router()
 const meteorService = require('../services/MeteorService')
+const MeteorDto = require('../dtos/MeteorDto')
 
-router.get('/', async (req, res, next) => {
+const dateValidator = require('../validators/DateValidator')
+const countValidator = require('../validators/CountValidator')
+const wereDangerousValidator = require('../validators/WereDangerousValidator')
+
+router.get('/', (req, res, next) => {
+    const quieryParams = req.query
+    const startDateParam = quieryParams.startDate
+    const endDateParam = quieryParams.endDate
+
+    dateValidator.validateStartDate(startDateParam)
```

Comment:
You can create a validator which will wrap all these calls and then call:
```
router.get('/', validator, async (req, res, next) => { 
    try {
        res.json(await meteorService.getMeteorsData())
        const requestParameters = req.query
        const meteorDto = new MeteorDto(requestParameters)
        res.json(await meteorService.getMeteorsData(meteorDto))
    } catch (error)
    ....
```
Also you can create a controller to wrap calls to service:
```
router.get('/', validator, controller.method);
```

Reply:
Great idea! Thanks
Fixed